systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaResource.java

```diff
@@ -185,11 +186,23 @@ private static KafkaBuilder defaultKafka(Kafka kafka, String name, int kafkaRepl
                         .withNewInlineLogging()
                             .addToLoggers("rootLogger.level", "DEBUG")
                         .endInlineLogging()
+                        .withReadinessProbe(
+                                new ProbeBuilder().withFailureThreshold(5).withInitialDelaySeconds(120).build()
+                        )
+                        .withLivenessProbe(
+                                new ProbeBuilder().withFailureThreshold(5).withInitialDelaySeconds(120).build()
+                        )
```

Comment:
You did IMHO not changed anything in User Operator. So this should not be here.

Reply:
Yeah ... but if I didn't prolong the delay here as well, it didn't go through ...

Reply:
The tests are passing completely fine outside of your PR. So there is no need to change this unless you changed something in the User Operator.

Reply:
This will definitely go out, just wanted to get a clean run of ST locally and CI ...

Reply:
Two of the testsets timed out - which could be also very well related to these changes since you probably increased the time needed to run the tests by more then one hour. So keep in mind that this might not always just help.